src/main/java/org/jabref/gui/JabRefDialog.java

```diff
@@ -54,6 +54,8 @@
     }
 
     private <T extends JabRefDialog> void trackDialogOpening(Class<T> clazz) {
-        Globals.getTelemetryClient().trackPageView(clazz.getName());
+        if (Globals.getTelemetryClient()!=null) {
```

Comment:
Checkstyle will hit you! 😈 

I think, this should be converted to an Optional, shouldn't it? Then the usual `Globals.getTelemtryClient().ifPresent(client -> client....)`